src/app-layer-htp-file.c

```diff
@@ -113,6 +113,13 @@ int HTPFileOpen(HtpState *s, const uint8_t *filename, uint16_t filename_len,
         }
 
         sbcfg = &s->cfg->response.sbcfg;
+        if (s->file_range_ids != 0) {
+            //TODO set an event ? or event better : handle intersected downloads
+            SCLogWarning(SC_ERR_NOT_SUPPORTED,
```

Comment:
Prints to screen based on traffic are not allowed. Guess an event to start?

Reply:
Can you point me to some similar code ? (generating an event for files)

Reply:
Ok, found some